tests/infra/testmode.py

```diff
@@ -0,0 +1,12 @@
+import os
+import unittest
+
+
+def standalone(f):
+    return unittest.skipIf(
```

Comment:
This is run if ^(A && B) == ^A || ^B

I think.  Demorgan's Law is so long long ago.

That means we run standalone if `INTEGRATION_TESTS not in os.environ or os.environ['INTEGRATION_TESTS'] == "0"`

Reply:
A test marked as `@standalone` will be skipped if condition is true.
<=> A test marked as `@standalone` will be run if condition is false.
<=> A test marked as `@standalone` will be run if `not (IT in env and $IT != "0")`
<=> A test marked as `@standalone` will be run if `IT not in env or $IT == "0"`

That's correct. 

If a test marked as `@standalone` is skipped, the message "Only running standalone tests" is printed. I don't think that's correct.